common/lib/capa/capa/responsetypes.py

```diff
@@ -2197,7 +2197,7 @@ def setup_response(self):
 
         # if <customresponse> has an "expect" (or "answer") attribute then save
         # that
-        self.expect = xml.get('expect') or xml.get('answer')
```

Comment:
@msegado expect is little confusing. can we have expected or something more meaningful ?